index.html

```diff
@@ -805,11 +803,30 @@ <h2>
                           </li>
                         </ol>
                       </li>
+                      <li>Let <var>identifier</var> be the value of the
+                      <a data-lt=
+                      "PaymentDetailsModifier.supportedMethods">supportedMethods</a>
+                      member of <var>modifier</var>.
+                      </li>
+                      <li>If required by the specification that defines the
+                      <var>identifier</var>, then <a data-cite=
+                      "WEBIDL#dfn-convert-ecmascript-to-idl-value">convert</a>
+                      <var>data</var> to an IDL value of the type specified
+                      there. Otherwise, <a data-cite=
+                      "WEBIDL#dfn-convert-ecmascript-to-idl-value">convert</a>
+                      to <a data-cite="WEBIDL#idl-object">object</a>. Rethrow
+                      any exceptions.
+                      </li>
+                      <li>If the user agent does not have an available
+                      <a>payment handler</a> for <var>identifer</var>,
+                      continue.
```

Comment:
We probably don't want to collect the unknown ones - only to convert them to retain current implementation behavior.